### PR TITLE
AKS updates to latest REST ContainerService specification

### PR DESCRIPTION
**Pull request by jskulavik**
## Description

---

This checklist is used to make sure that common guidelines for a pull request are followed.
- [X] Please add REST spec PR link to the SDK PR
- [X] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [X] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [X] Title of the pull request is clear and informative.
- [X] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [X] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [X] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [X] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [ ] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

There was an update to the [REST API Specification PR](https://github.com/Azure/azure-rest-api-specs/pull/4896) that was a predecessor to this PR.

### CODE REVIEW

**Comment by dsgouda** [comment]
Please fix the failing test cases

File: src/SDKs/ContainerService/ContainerService.Tests/Microsoft.Azure.Management.ContainerService.Tests.csproj [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
```
Comment (dsgouda) [inline src/SDKs/ContainerService/ContainerService.Tests/Microsoft.Azure.Management.ContainerService.Tests.csproj]: Please insert this at line 2
```
<Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.test.reference.props'))" />
```

File: src/SDKs/ContainerService/ContainerService.Tests/Microsoft.Azure.Management.ContainerService.Tests.csproj [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
+
+    <IsPackable>false</IsPackable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Azure.Management.ResourceManager" Version="2.0.0-preview" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.9.0" />
+    <PackageReference Include="Microsoft.Rest.ClientRuntime.Azure.TestFramework" Version="1.7.5" />
```
Comment (dsgouda) [inline src/SDKs/ContainerService/ContainerService.Tests/Microsoft.Azure.Management.ContainerService.Tests.csproj]: Please remove these refs.
Please use [this](https://github.com/Azure/azure-sdk-for-net/blob/psSdkJson6/src/SDKs/Advisor/Advisor.Tests/Advisor.Tests.csproj) as a template for the test csproj

File: src/SDKs/ContainerService/ContainerService.Tests/Microsoft.Azure.Management.ContainerService.Tests.csproj [hunk 3]
```diff
@@ -0,0 +1,21 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
```
Comment (dsgouda) [inline src/SDKs/ContainerService/ContainerService.Tests/Microsoft.Azure.Management.ContainerService.Tests.csproj]: Please remove lines 3-7

File: src/SDKs/ContainerService/Management.ContainerService/Microsoft.Azure.Management.ContainerService.csproj [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
+  <!-- Please do not move/edit code below this line -->
+  
+  <PropertyGroup>
+    <PackageId>Microsoft.Azure.Management.ContainerService</PackageId>
+    <Description>Provides developers with a library to create and manage all Azure Kubernetes Service resources. </Description>
+    <Version>1.0.0</Version>
+    <AssemblyName>Microsoft.Azure.Management.ContainerService</AssemblyName>
+    <PackageTags>Microsoft Azure Kubernetes Service; Microsoft Azure ContainerService;</PackageTags>
+    <PackageReleaseNotes></PackageReleaseNotes>
```
Comment (dsgouda) [inline src/SDKs/ContainerService/Management.ContainerService/Microsoft.Azure.Management.ContainerService.csproj]: Please add `PackageReleaseNotes`
An example [here](https://github.com/Azure/azure-sdk-for-net/blob/psSdkJson6/src/SDKs/CognitiveServices/dataPlane/Vision/ContentModerator/ContentModerator/Microsoft.Azure.CognitiveServices.ContentModerator.csproj#L12)

File: src/SDKs/ContainerService/Management.ContainerService/Microsoft.Azure.Management.ContainerService.csproj [hunk 5]
```diff
@@ -0,0 +1,24 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <!-- Please do not move/edit code below this line -->
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
+  <!-- Please do not move/edit code below this line -->
+  
+  <PropertyGroup>
+    <PackageId>Microsoft.Azure.Management.ContainerService</PackageId>
+    <Description>Provides developers with a library to create and manage all Azure Kubernetes Service resources. </Description>
+    <Version>1.0.0</Version>
```
Comment (dsgouda) [inline src/SDKs/ContainerService/Management.ContainerService/Microsoft.Azure.Management.ContainerService.csproj]: Just to confirm, you are planning to start off with a stable version or are you planning initial preview versions before releasing a stable version?

**Review by dsgouda — CHANGES_REQUESTED** [review]

File: src/SDKs/ContainerService/Management.ContainerService/Microsoft.Azure.Management.ContainerService.csproj (hunk 5 above)
Comment (jskulavik) [reply src/SDKs/ContainerService/Management.ContainerService/Microsoft.Azure.Management.ContainerService.csproj]: We are targeting the stable version at this time.

**Review by shahabhijeet — CHANGES_REQUESTED** [review]
Please cleanup your commit history and then this PR will be reviewed/merged.
Have logical set of commits, currently this PR has way too many commits and needs to be cleaned up.

**Pull request closed without merging**